bin/desi_tile_vi

```diff
@@ -136,6 +137,8 @@ def main():
 
     args=parse()
 
+    archivedate = datetime.datetime.now().strftime('%Y%m%d')
```

Comment:
Do we need to concern ourselves with timezones or will these always be used solely at NERSC in pacific times?

Reply:
Good question, but I don't think we need to worry about timezones.  I think the place where it would matter would be if collaborators in China or Australia mirrored the data and then did tile_vi from there for daily operations, which seems unlikely.  If someone disagrees, I invite them to contribute the timezone robust code.